docs/csharp/language-reference/compiler-options/refonly-compiler-option.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "-refonly (C# Compiler Options)"
+ms.date: "2017-07-08"
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+f1_keywords: 
+  - "/refonly"
+dev_langs: 
+  - "CSharp"
+helpviewer_keywords: 
+  - "/refonly compiler option [C#]"
+  - "-refonly compiler option [C#]"
+author: "BillWagner"
+ms.author: "wiwagn"
+---
+
+# /refonly (C# Compiler Options)
+
+The **/refonly** The /refonly option indicates that a ref assembly should be output instead of an implementation assembly, as the primary output. The `/refonly` parameter silently disables outputting PDBs, as ref assemblies cannot be executed.
+
+## Syntax
+
+```console
+/refonly
+```
+
+## Remarks
+
+Metadata-only assembly have their method bodies replaced with a single `throw null` body, but include all members except anonymous types. The reason for using `throw null` bodies (as opposed to no bodies) is so that PEVerify could run and pass (thus validating the completeness of the metadata).
+
+Ref assemblies include an assembly-level `ReferenceAssembly` attribute. This attribute may be specified in source (then we won't need to synthesize it). Because of this attribute, runtimes will refuse to load ref assemblies for execution (but they can still be loaded in reflection-only mode). Some tools may be affected and will need to be updated (for example, `sgen.exe`).
```

Comment:
you should avoid the use of we, unless when we (Microsoft) is recommending something. so who is we in this case?

avoid future tense if possible